### DISCUSSION TITLE
Liquid Soap file refresh when live source disconnects

### DISCUSSION THREAD

**Discussion by MayhemOffTopiC** [AzuraCast/AzuraCast#3883] [Q&A]
I am wondering if there is a way to configure liquid soap to skip to a new song when a live source disconnects from the server.
At the moment, when a dj connects, the current song being played on auto dj cuts out. That is fine but as soon as the live source disconnects, the song picks up right where it got cut off. It is annoying and sounds bad

**Comment by Vaalyn** [comment]
It should be possible to let Liquidsoap skip the song it was playing when a DJ connects.

This would need to be investigated and made into a configurable option in AzuraCast so that users can choose if they want this or not.

For reference regarding skipping songs in Liquidsoap see https://github.com/savonet/liquidsoap/issues/530

**Comment by YosefAdPro** [comment]
It will also improve this matter

https://github.com/AzuraCast/AzuraCast/issues/3492#issuecomment-737403677
